### PR TITLE
feat: delegations alliance assets routing

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

**Review by restorenode — CHANGES_REQUESTED** [review]
https://github.com/terra-money/station/assets/30275692/cd3d7d21-1f7a-44c4-b589-e4ec1625104c

**Comment by ericHgorski** [comment]
> validator-page-error.mov

Whoops had wrong args order. Should be fixed now.

File: src/components/general/StakingAssetLink.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </InlineFlex>
+      </InternalLink>
+    ) : (
+      <ValidatorLink address={address} internal img />
+    )
+  }
+  return render()
+}
+
+export default StakingAssetLink
```
Comment (restorenode) [inline src/components/general/StakingAssetLink.tsx:40]: I would reformat the `ValidatorLink` component in order to take on this functionality.  `StakingAssetLink` seems to take on most of the same code.

**Review by restorenode — DISMISSED** [review]

File: src/components/general/StakingAssetLink.tsx (hunk 1 above)
Comment (ericHgorski) [reply src/components/general/StakingAssetLink.tsx:40]: I explored that route initially but felt it added too much complexity to a widely used component.

**Review by restorenode — APPROVED** [review]

**Pull request merged**